### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by mkaput**

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      interval: weekly
```
Comment (piotmag769) [inline .github/dependabot.yml:11]: monthly maybe?

Comment (mkaput) [reply .github/dependabot.yml:11]: we use weekly in Scarb and it works

Comment (Arcticae) [reply .github/dependabot.yml:11]: nobody wants to deal with that every week :/

Comment (mkaput) [reply .github/dependabot.yml]: done

File: .github/dependabot.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: npm
```
Comment (piotmag769) [inline .github/dependabot.yml:8]: Can we group them? E.g. https://github.com/software-mansion/cairo-profiler/blob/main/.github/dependabot.yml, so it is all in one PR

Comment (mkaput) [reply .github/dependabot.yml:8]: yes! I forgot about this feature

File: .github/dependabot.yml [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+      interval: monthly
+    groups:
+      all:
+        patterns:
+          - "*"
+
+  - package-ecosystem: github-actions
+    directory: "/"
+    schedule:
+      interval: monthly
```
Comment (piotmag769) [inline .github/dependabot.yml:20]: Group here as well?

**Pull request merged**